commit/metrics/prom.go

```diff
@@ -46,6 +49,22 @@ var (
 		},
 		[]string{"chainID", "state", "type"},
 	)
+	promMerkleProcessorRmnReportLatency = promauto.NewHistogramVec(
+		prometheus.HistogramOpts{
+			Name:    "ccip_commit_merkle_processor_rmn_report_latency",
+			Help:    "This metric tracks the client-observed latency of building an full RMN report with signatures",
+			Buckets: prometheus.DefBuckets,
+		},
+		[]string{"dest_chain_selector", "success"},
```

Comment:
I would consider transforming `dest_chain_selector` to the destChainID. DestChainSelectors are very hard to reason about - they require an additional step to transform to chainID which are human-readable. Also other metrics rely on chain ids instead of selectors